Some Void systems, inluding an installation from the live image, do not have an
Xsession script where upstream expects it to be, so use the fallback Xsession
script provided by the Void tigervnc package.

--- a/unix/vncserver/vncserver.in	2023-02-28 10:11:20.000000000 -0500
+++ b/unix/vncserver/vncserver.in	2023-07-30 17:55:47.604000000 -0400
@@ -439,7 +439,7 @@
 	die "$prog: couldn't find \"$cmd\" on your PATH.\n";
     }
 
-    foreach $cmd ("/etc/X11/xinit/Xsession", "/etc/X11/Xsession") {
+    foreach $cmd ("/etc/X11/xinit/Xsession", "/etc/X11/Xsession", "/etc/tigervnc/Xsession-fallback") {
         if (-x "$cmd") {
             $Xsession = $cmd;
             last;
